Dockerfile

```diff
@@ -158,44 +178,46 @@ RUN rm /etc/apache2/sites-enabled/000-default.conf
 COPY --from=builder /tmp/local/ /opt/perl/local/
 ENV PERL5LIB="/opt/product-opener/lib/:/opt/perl/local/lib/perl5/"
 ENV PATH="/opt/perl/local/bin:${PATH}"
+# Create writable dirs and change ownership to www-data
+RUN \
+    mkdir -p var/run/apache2/ && \
+    chown www-data:www-data var/run/apache2/ && \
+    for path in users products product_images orgs new_images logs; do \
+        mkdir -p /mnt/podata/${path}; \
+        chown www-data:www-data /mnt/podata/${path}; \
+    done && \
+    # Create symlinks of data files in /mnt/podata (because we currently mix data and conf data)
+    for path in ecoscore emb_codes forest-footprint ingredients lang packager-codes po taxonomies templates; do \
+        ln -sfT /opt/product-opener/${path} /mnt/podata/${path}; \
+    done
 
 EXPOSE 80
 COPY ./docker/docker-entrypoint.sh /
+WORKDIR /opt/product-opener/
+USER www-data
 ENTRYPOINT [ "/docker-entrypoint.sh" ]
+# default command is apache2ctl start
 CMD ["apache2ctl", "-D", "FOREGROUND"]
 
-FROM runnable AS runnable-vscode
+######################
+# Image for dev
+######################
+# note: user is not root but you car run --user root in docker if you want
+FROM runnable AS vscode
```

Comment:
do we need the vscode section at all ? Are you using it ? I'm using vscode but find no need for it.